way_to_home/tests/unittests/place/testmodel.py

```diff
@@ -0,0 +1,110 @@
+"""
+Place model tests
+===============
+"""
+from decimal import Decimal
+
+from custom_user.models import CustomUser
+from django.test import TestCase, Client
+
+from place.models import Place
+
+
+class PlaceModelTest(TestCase):
+    """TestCase for providing Place model testing"""
+    def setUp(self):
+        """Method that provides preparation before testing Place model features."""
+        custom_user = CustomUser.objects.create(id=2, email='mymail@icloud.com', is_active=True)
+        custom_user.set_password('qwerty12345')
+        custom_user.save()
+
+        self.client = Client()
+        self.client.login(email='mymail@icloud.com', password='qwerty12345')
```

Comment:
You don't use `client` instance in this module

Reply:
right